tldw_Server_API/tests/MediaIngestion_NEW/unit/test_persistence_chunk_consistency.py

```
@@ -30,6 +31,78 @@ def observe(self, *_args: Any, **_kwargs: Any) -> None:
         return None
 
 
+class _RepoBackedWorkerDB:
+    instances: list["_RepoBackedWorkerDB"] = []
```

Ruff (0.15.6)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)